apps/super-app/src/common/components/support-detail-page/SupportDetailPage.tsx

```diff
@@ -214,6 +214,7 @@ export function SupportDetailPage({
         blurbIntro={blurbIntro}
         blurbFacebookLinks={blurbFacebookLinks}
       ></DescriptionSection>
+      <Carousel carouselSection={carouselSection}></Carousel>
```

Comment:
unfortunately the carousel element still renders for partners who have an empty `carouselSection`, which leads to some content on the bottom being blocked out:
![Screenshot 2023-12-31 at 10 33 02 PM](https://github.com/bettersg/call-home/assets/41856541/e76d643f-3d59-447f-a68a-334371a63748)
perhaps we could try to avoid rendering it unnecessarily by making `carouselSection` nullable, or instead just doing something like this?
```suggestion
      { carouselSection.length > 0 && <Carousel carouselSection={carouselSection} /> }
```